metric/holder.go

```diff
@@ -0,0 +1,271 @@
+package metric
+
+import (
+	"sync"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const genTimeInterval = time.Second
+
+var (
+	nowTime   time.Time
```

Comment:
Can we use atomic.Int64 here?

```go
var nowTime = atomic.Int64{}

func setNowTime(t time.Time) {
  nowTime = t.UnixNano()
}

func getNowTime() time.Time {
  return time.Unix(0, nowTime.Load())
}
```

Reply:
I think we can move this logic to a new package `github.com/ozontech/file.d/xtime.GetInaccurateTime()`